packages/account-kit/src/steps/finalizing/FinalizingStep.tsx

```diff
@@ -32,11 +33,18 @@ export function FinalizingStep() {
       if (!appAccountClient) throw new Error("App account client not ready.");
       if (!registerDelegationSignature) throw new Error("No delegation signature.");
 
+      // TODO: should this use `callWithSignature`?
```

Comment:
It does a few lines below?

Reply:
yeah i meant the `callWithSignature` util we have (that i believe we use for `registerSpender`)

Reply:
ohhh I see! that function is built to request the signature so wasn't immediately available for use here could be adapted